example/example.py

```diff
@@ -48,11 +48,14 @@
 # ---------------------------------------------------------------------- TYPES & QUERIES
 
 
-class Message(
```

Comment:
Why? What is the purpose of these changes?

Reply:
This, (and the other ones similar to this) were to fix mypy on Travis.

https://travis-ci.org/github/datadvance/DjangoChannelsGraphqlWs/jobs/630643421?utm_medium=notification&utm_source=github_status

```
>                   stderr)
E               plumbum.commands.processes.ProcessExecutionError: Unexpected exit code: 1
E               Command line: | /home/travis/virtualenv/python3.7.1/bin/mypy example/
E               Stdout:       | example/example.py:51: error: Unexpected keyword argument "default_resolver" for "__init_subclass__" of "object"
```

![image](https://user-images.githubusercontent.com/26336/78461450-a1310780-768e-11ea-8ae3-ecb8478f6795.png)


(Also, pardon me!): I'm just not familiar with the use of classes like this.  What is `default_resolver=X` doing from a python perspective?  Being passed along into `__init_subclass__`? Is there a place in the python documentation for how this works or what it's called?